packages/client-core/src/user/components/Wallet/wallet.tsx

```diff
@@ -33,8 +33,8 @@ export const WalletPage = (): any => {
     type,
     coinlimit,
     isLoading,
-    walletDataReceive,
-    walletData,
+    //walletDataReceive,
```

Comment:
please remove all code that has been commented out

Reply:
Changed in current version